tests/test_apis_validate.py

```diff
@@ -19,4 +19,7 @@ def test_validate_definitions(self):
             for defintion_file in alsdkdefs.get_service_defs(service):
                 print("Validating def", defintion_file)
                 obj = alsdkdefs.get_spec(alsdkdefs.make_file_uri(defintion_file))
-                alsdkdefs.validate(obj, defintion_file)
+                try:
+                    alsdkdefs.validate(obj, defintion_file)
+                except alsdkdefs.AlertLogicOpenApiValidationWarning as e:
+                    print(f"***WARNING***: definition may not meet criteria {e}")
```

Comment:
Is the a unit test, or the actual execution? I'm confused about what's being tested here.

Reply:
it is unit test, but we don't want to fail it in case of Warning exception.